views/matrixsplit.html

```diff
@@ -233,8 +233,8 @@
             =
             \begin{bmatrix}
```

Comment:
yeah that seems fine, too (mathematic is sneakily cheap: it's free if you have a raspberry pi, which itself is about $35)